README.rst

```diff
@@ -0,0 +1,75 @@
+Linux Integration Services Automation (LISA)
```

Comment:
So we'll keep both versions now? Why? What about maintaining the two sets?

Reply:
The doc website is used to browse. The README.rst is the fallback of document, so it's all links go to doc site. Other files are doc source files.